examples/add_volume_series.py

```diff
@@ -0,0 +1,18 @@
+"""
+Display one 3-D volume layer using the add_volume API
```

Comment:
can you rename this example `nD_volume` and adjust the doc string to say you are using `4D` data representing a volumetric timeseries.